### ISSUE TITLE
Set-PSFConfig should accept pipeline Input

### ISSUE THREAD

**Issue by figueroadavid** [PowershellFrameworkCollective/psframework#400]
The Set-PSFConfig should allow pipeline input. The primary reason for this is being able to do something like:
```powershell
Get-PSFConfig -name <module.config.setting> | set-PSFConfig -value <new value>
```

**Comment by FriedrichWeinmann** [comment]
Thanks for bringing that up.
Added to the list of things to include in the next version

**Comment by FriedrichWeinmann** [comment]
Released in 1.4.146
